src/lib/config_v2/config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sync"
+	"time"
+
+	"gopkg.in/yaml.v3"
+)
+
+//Config struct stores the APIs parsed from local-config.yml file.
+type Config struct {
+	path string
+	mux  sync.RWMutex
+	APIs []API `yaml:"apis"`
+}
+
+//API stores the information of the endpoints.
+type API struct {
+	Name    string            `yaml:"name,omitempty"`
+	Every   time.Duration     `yaml:"every,omitempty"`
+	Domain  string            `yaml:"domain_or_ip,omitempty"`
+	Route   string            `yaml:"route,omitempty"`
+	Method  string            `yaml:"method,omitempty"`
+	Headers map[string]string `yaml:"headers,omitempty"`
+	Params  map[string]string `yaml:"params,omitempty"`
+	Body    map[string]string `yaml:"body,omitempty"`
+}
+
+//Instantiate a Config object and reloads the data from the given file location.
```

Comment:
```suggestion
// New returns a new configuration after reloading from the given path.
```

Kindly follow previous suggestions please!